api/v1beta1/watcherapi_types.go

```diff
@@ -35,6 +35,13 @@ type WatcherAPISpec struct {
 
 	WatcherSubCrsCommon `json:",inline"`
 
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=1
+	// +kubebuilder:validation:Maximum=32
```

Comment:
maximum 1 here?

[EDIT] no... this is api probably this one is safe to run more than one instance of it

Reply:
yep, we can have multiple replicas for the api without any problem